pillar.example

```diff
@@ -8,6 +8,8 @@ hana:
       sid: 'prd'
       instance: 00
       password: 'Qwerty1234'
+      # saptune solution to apply ( by default nothing is applied)
+      saptune_solution: 'MAXDB'
```

Comment:
Set `HANA` in the example. It makes more sense as this formula is about HANA

Reply:
done thc